tests/unittest.cpp

```diff
@@ -524,9 +526,9 @@ auto createCollections(const size_t nElements = 3u) {
 }
 
 // Helper functionality to keep the tests below with a common setup a bit shorter
-void checkCollections(const ExampleHitCollection& hits, const ExampleClusterCollection& clusters,
-                      const ExampleWithVectorMemberCollection& vectors,
-                      const podio::UserDataCollection<float>& userData, const size_t nElements = 3u) {
+void checkCollections(/*const*/ ExampleHitCollection& hits, /*const*/ ExampleClusterCollection& clusters,
+                      /*const*/ ExampleWithVectorMemberCollection& vectors,
+                      /*const*/ podio::UserDataCollection<float>& userData, /*const*/ size_t nElements = 3u) {
```

Comment:
These should all be `const`, but currently `prepareForWrite` is not thread-safe and thus not marked `const`.

Reply:
Once #295 is merged, these should be possible as `const` again.

Reply:
`getBuffers` is still not marked `const` preventing this. Fixing that in a separate PR.

Reply:
Tracked in #302